blockchain/blockdao/blockdao.go

```diff
@@ -57,11 +57,51 @@ type (
 		footerCache  *cache.ThreadSafeLruCache
 		tipHeight    uint64
 	}
+	// ModuleConfig represents the configuration of File DAO
+	ModuleConfig struct {
```

Comment:
maybe name it `BlockdaoConfig`, in the future we'll have similar config for other modules